Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/ui/Account/CreateAccServlet.java

```diff
@@ -0,0 +1,40 @@
+package ui.Account;
+
+import services.UserService;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+public class CreateAccServlet extends HttpServlet {
```

Comment:
Another important point. I will not mention all similar errors, so change through all the codebase